extension/pom.xml

```diff
@@ -45,16 +55,45 @@
 
     <!-- Test scope-->
     <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+    </dependency>
```

Comment:
🔧 It mentions this dependency under test scope, but the scope of is not test. I don't think it should be test as this is in the extension, but I see there is also a new dependency `junit-platform-commons`. I'm not sure what this one  is.